server/src/main/java/org/opensearch/action/admin/cluster/node/tasks/create/CreateTaskAction.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.node.tasks.create;
+
+import org.opensearch.action.ActionType;
+
+/**
+ * Transport action for creating new task.
+ *
+ * @opensearch.internal
+ */
+public class CreateTaskAction extends ActionType<CreateTaskResponse> {
```

Comment:
how are tasks being created today?

Reply:
Task is created when we any transport action invoked.